src/flipNumbers.js

```diff
@@ -5,6 +5,8 @@ import FlipNumber from './flipNumber';
 type Props = {
   numbers: string | Array<string>,
   nonNumberStyle?: { [string]: string | number },
```

Comment:
could you fix this one as well?

Reply:
Do you mean the misnamed prop? If so both `numberStyle` and `nonNumberStyle` are fixed in this pr. If not could you clarify?

Reply:
Hi, just fix this `nonNumberStyle?: React.CSSProperites,`. thanks for the PR 👍

Reply:
Fixed this and one other.